aws/table_aws_vpc_verified_acces_group.go

```diff
@@ -0,0 +1,195 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsVpcVerifiedAccessGroup(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_vpc_verified_access_group",
+		Description: "AWS VPC Verified Access Group",
+		List: &plugin.ListConfig{
+			Hydrate: listVpcVerifiedAccessGroups,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidParameterValue"}),
+			},
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "verified_access_group_id", Require: plugin.Optional},
+				{Name: "verified_access_instance_id", Require: plugin.Optional},
+			},
+		},
+		GetMatrixItemFunc: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "verified_access_group_id",
+				Description: "The ID of the Verified Access group.",
```

Comment:
```suggestion
				Description: "The ID of the verified access group.",
```

I believe all the instances of the `verified access group` can be updated as per the above suggestion.